contentcuration/contentcuration/assets/channelList/views/ChannelListIndex.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <VApp>
```

Comment:
Here are the color names I'm using for vuetify:
https://github.com/jayoshih/content-curation/blob/vue-edit/contentcuration/contentcuration/static/js/bundle_modules/sandbox.js#L17

Not too much there yet, but I'm making blue the primary color since it gets used most frequently

Reply:
We need to unify the vuetifies!